EventHandlers/Game.js

```diff
@@ -19,9 +20,98 @@ function playerJoiningGame(gameIdstr) {
   const playerID = this.socket.id;
   const activeGame = this.games.find((game) => game.id === gameId);
   if (activeGame.players) {
-    activeGame.players.push(playerID);
+      activeGame.players.push(playerID);
   } else {
     activeGame.players = [playerID];
   }
   console.log(this.games);
 }
+
+function rowLabel(row) {
+    if (row == 1) return "A";
+    if (row == 2) return "B";
+    if (row == 3) return "C";
+    if (row == 4) return "D";
+    if (row == 5) return "E";
+    if (row == 6) return "F";
+    if (row == 7) return "G";
+    if (row == 8) return "H";
+    if (row == 9) return "I"; 
+}
+function playerStartsGame(gameIdstr) {
+    //Tiles is a 11 rows by 14 columns character array
+    //Rows 0 and 10 are edges, Columns 0 and 13 are edges
+    //Each tile is one of the following:
+    //E - edge
+    //B - bag
+    //X - non-chain tile on board
+    //#n - tile in player n's tray
+    //A,C,F,I,L,T,W - chain letter designation (as chains are formed)
+
+    const gameId = Number.parseInt(gameIdstr);
+    const activeGame = this.games.find((game) => game.id === gameId);
+    const playerID = this.socket.id;
+    console.log("Number of players ... " + activeGame.players.length);
+
+    var tiles = Array(11);
+    for (var i = 0; i <=10; i++) {
+        tiles[i] = new Array(14);
+    }
+    //clear the board 'B' : Empty board piece
+    for (let i = 1; i <= 10; i++)
+        for (let j = 1; j <= 12; j++) {
+           tiles[i][j] = "B";
+        }
+    //assign top/bottom edge tiles labeled 'E'
+    for (let i = 0; i <= 13; i++) {
+        tiles[0][i] = "E";
+        tiles[10][i] = "E";
+    }
+    //assign left/right edge tiles labeled 'E'
+    for (let i = 0; i <= 10; i++) {
+        tiles[i][0] = "E";
+        tiles[i][13] = "E";
+    }
+ 
+    var numberOfPlayers = 4;
```

Comment:
This is hard coded to 4 players, it would be cool to be able to play with more and less players then 4